### PR TITLE
Stats: Removing released feature flag

### PR DESCRIPTION
**Pull request by grzegorz-cp**
Related to #73674

## Proposed Changes

* removing feature flag for the Insights page grid that was released few weeks ago

## Testing Instructions

* navigate to the `Insights` page on the live branch
* verify that the grid loads properly and all modules look good
* test across different breakpoints

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/stats/grid-layout.scss [hunk 1]
```diff
@@ -129,27 +129,6 @@ $grid-vertical-gutters: 32px;
...
-				}
-			}
-
-			.card.stats-module {
-				margin-bottom: 0;
-			}
-		}
-	}
-
 	.stats__module-wrapper--tags-categories,
```
Comment (dognose24) [inline client/my-sites/stats/grid-layout.scss:132]:
```suggestion
```

The class name `stats__module-wrapper--tags-categories` seems to be unused anymore.

**Review by dognose24 — APPROVED** [review]
Works as described! I found styles for the class name that may not be used anymore, which could be removed to clean up the naming.

**Review by jsnmoon — APPROVED** [review]
The code looks good, and the page continues to work as expected. Nice work!

File: client/my-sites/stats/grid-layout.scss (hunk 1 above)
Comment (grzegorz-cp) [reply client/my-sites/stats/grid-layout.scss:132]: Great observation @dognose24. There are more classes like this one and I can remove them when working on [#75742](https://github.com/Automattic/wp-calypso/issues/75742). I'll add it as one of the requirements of the other issue.

**Pull request merged**